src/spikeinterface/qualitymetrics/tests/conftest.py

```diff
@@ -35,3 +37,52 @@ def _small_sorting_analyzer():
 @pytest.fixture(scope="module")
 def small_sorting_analyzer():
     return _small_sorting_analyzer()
+
+
+def _sorting_analyzer_simple():
+
+    # we need high firing rate for amplitude_cutoff
+    recording, sorting = generate_ground_truth_recording(
+        durations=[
+            120.0,
+        ],
+        sampling_frequency=30_000.0,
+        num_channels=6,
+        num_units=10,
+        generate_sorting_kwargs=dict(firing_rates=10.0, refractory_period_ms=4.0),
+        generate_unit_locations_kwargs=dict(
+            margin_um=5.0,
+            minimum_z=5.0,
+            maximum_z=20.0,
+        ),
+        generate_templates_kwargs=dict(
+            unit_params=dict(
+                alpha=(200.0, 500.0),
+            )
+        ),
+        noise_kwargs=dict(noise_levels=5.0, strategy="tile_pregenerated"),
+        seed=1205,
+    )
+
+    sorting_analyzer = create_sorting_analyzer(sorting, recording, format="memory", sparse=True)
+
+    sorting_analyzer.compute("random_spikes", max_spikes_per_unit=300, seed=1205)
+    sorting_analyzer.compute("noise_levels")
+    sorting_analyzer.compute("waveforms", **job_kwargs)
+    sorting_analyzer.compute("templates")
+    sorting_analyzer.compute("spike_amplitudes", **job_kwargs)
+
+    return sorting_analyzer
+
+
+@pytest.fixture(scope="module")
+def sorting_analyzer_simple():
+    sorting_analyzer = get_sorting_analyzer(seed=2205)
+    return sorting_analyzer
+
+    return sorting_analyzer
+
+
+@pytest.fixture(scope="module")
```

Comment:
I believe we had session before for that very reason, but then it got changed in place, which then caused code-cov to fail because the full test session was using the same analyzer. So at that time we took the time hit to ensure safety because it was super super hard to figure out which analyzer was causing the full test problems.

Reply:
I think we should stick with session, since the `analyzer` is getting modified in place when we calculate quality metrics. Seems safer?

Reply:
That makes sense!